racket/src/cs/core/control.ss

```diff
@@ -344,8 +371,11 @@
 
 ;; ----------------------------------------
 
+
+;; This could block because prompt tag could be past future tag
```

Comment:
This comment would apply all over and isn't the most important things about `abort-current-continuation`. I think that using a descriptive name like `maybe-future-barricade` makes the comment unnecessary.